ci/release/update-version.sh

```diff
@@ -16,6 +16,4 @@ function sed_runner() {
 }
 
 # Centralized version file update
-echo "${NEXT_FULL_TAG}" > pynvjitlink/VERSION
-sed_runner 's/'"^  VERSION [0-9\.]*"'/'"  VERSION ${NEXT_FULL_TAG}"'/g' CMakeLists.txt
 sed_runner 's/^version = "[0-9\.]*"/version = "'${NEXT_FULL_TAG}'"/g' pyproject.toml
```

Comment:
As the version now lives in one place, maybe this script isn't needed?

Left it for now as it could be ergonomic when working across projects. Though open to other thoughts here

Reply:
Please leave this script. It helps a lot to have a single reference for how versions are used in every RAPIDS-related project.

Reply:
Yep it is still there 🙂

Made the same observation above